### ISSUE TITLE
Cognito Accounts

### ISSUE THREAD

**Issue by osarhan** [Apollon77/alexa-cookie#136]
**Is your feature request related to a problem? Please describe.**
In order to get a token, it requires someones entire amazon account access login. 

**Describe the solution you'd like**
A cookie that is a bit more secure, and dedicated to alexa controls.

**Describe alternatives you've considered**
Possibly be able to control someones alexa using a cognito user account, or alexa account linking. 

**Additional context**
I am trying to provide a service for alexa users to trigger certain actions on different times everyday, managed by my cloud scheduler. Specifically its to call the muslim athan (call to prayer).  Since these times change everyday, users cant just set up routines.  I am hoping to build a onestop shop solution to help trigger alexas, and since alexa skills dont allow proactive events to trigger skills or audio, this is potentially the perfect solution. 

TLDR: can I use someones cognito account/alexa account linking token to use the alexa remote control shell script?

**Comment by Apollon77** [comment]
I do not think so. Sorry
